R/01-etl.R

```diff
@@ -53,8 +53,10 @@ report.default <- function(x, ...) {
   updated <- data.frame(
     tableName = x$name,
```

Comment:
@jan-abel-inwt Could you check if the column names match to those on the server? Thanks!

Reply:
![image](https://user-images.githubusercontent.com/106665518/225067636-1ed97fc3-6c9c-4020-ba2a-97372306583b.png)